addons/mail/static/src/components/attachment_link_preview/attachment_link_preview.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.AttachmentLinkPreview" owl="1">
+        <div class="o_AttachmentLinkPreview" t-att-class="{'o-isCompact': props.isCompact}">
+            <t t-if="attachmentLinkPreview and attachmentLinkPreview.attachment">
+                <t t-if="attachmentLinkPreview.imageUrl">
+                    <img t-att-src="attachmentLinkPreview.imageUrl" class="o_AttachmentLinkPreview_linkImage"/>
+                </t>
+            <div class="o_AttachmentLinkPreview_aside">
+                    <button class="o_AttachmentLinkPreview_asideItem o_AttachmentLinkPreview_asideItemUnlink" t-att-class="{ 'o-pretty': attachmentLinkPreview.attachment.isLinkedToComposer }" t-on-click="attachmentLinkPreview.onClickUnlink">
+                        <i class="fa fa-trash"/>
+                    </button>
+                </div>
+                <t t-if="attachmentLinkPreview.attachment.name">
+                    <div class="o_AttachmentLinkPreview_title">
+                        <a t-att-href="attachmentLinkPreview.attachment.url">
+                            <h3 class="o_AttachmentLinkPreview_Linktitle">
+                                <t t-esc="attachmentLinkPreview.attachment.name"/>
+                            </h3>
+                        </a>
+                    </div>
+                </t>
+                <t t-if="attachmentLinkPreview.attachment.description">
+                    <p class="o_AttachmentLinkPreview_excerpt">
```

Comment:
name the part description like the field name,
could have the t-if on the p to simplify the template a bit (same for other conditions in here)